hooks/pre/30-upgrade.rb

```diff
@@ -1,3 +1,13 @@
+def stop_services
+  Kafo::Helpers.execute('katello-service stop')
+end
+
+def start_mongo
+  status = Kafo::Helpers.execute('service-wait mongod status')
+  Kafo::Helpers.execute('service-wait mongod start') unless status
```

Comment:
So I exclude mongod from the stop now, so in MOST cases it will already be running by the time we check here. If it's not, and we hit this timeout problem, the pulp_migrate will fail, and the installer will exit, so at least it's not silent.